react/src/queries/fragments.ts

```diff
@@ -275,6 +275,10 @@ export const SHIPMENT_FIELDS_FRAGMENT = gql`
     sentBy {
       ...UserBasicFields
     }
+    receivingStartedOn
```

Comment:
fyi @pylipp we did not update the mock for test 4.4.1.4. Therefore, the test fails now.

I pushed a quick fix on master

Reply:
Ooh thanks :+1: